migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc

```diff
@@ -6,9 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the {mtc-first}. The {mtc} enables you to control the migration and to minimize application downtime.
```

Comment:
{mtc-first} = Cluster Application Migration (CAM) tool
{mtc} = CAM tool [OpenShift 3 to 4]